HomeOxygenTherapy/R4/files/HomeOxygenTherapyPrepopulation-0.1.0.cql

```diff
@@ -82,37 +58,41 @@ concept "Intent Original Order": { "Intent Original Order code" } display 'Origi
 
 parameter device_request DeviceRequest
 
-// Relevant Diagnosis Code Lists
-define "Pulmonary_hypertension_Codes": { "I27.0", "I27.2" }
-
 // Observation code lists
+
+//Arterial Oxygen Codes
 define "Arterial_oxygen_saturation_Codes": { "59408-5" }
 define "Arterial_partial_pressure_of_oxygen_Codes": { "2703-7" }
 define "Arterial_oxygen_saturation_during_exercise_Codes": { "89276-0" }
 
-define "Hematocrit_lab_test_Codes": { "20570-8", "31100-1", "32354-3", "41654-5", "41655-2", "4544-3", "4545-0", "71829-6", "71830-4", "71832-0", "71833-8" }
-define "Hematocrit_Codes": {"31100-1"}
+//Hematocrit Code
+define "Hematocrit_lab_test_Codes": {"32354-3"}
+
 // Other codes
-define "Immobilization_Codes": { "102491009", "129857008", "129859006", "160685001", "371632003", "8510008" }
+define "Immobilization_Codes": { "Immobilization" }
```

Comment:
You Don't need to assign the Immobilization value set to a variable in this line.

Below, when you define "PatientMobile", replace what you have with:
define PatientMobile: not exists(Confirmed(ActiveOrRecurring([Condition: "Immobilization"])))